src/main/java/de/unijena/cheminf/mortar/model/fragmentation/FragmentationService.java

```diff
@@ -24,10 +24,7 @@
 import de.unijena.cheminf.mortar.message.Message;
 import de.unijena.cheminf.mortar.model.data.FragmentDataModel;
 import de.unijena.cheminf.mortar.model.data.MoleculeDataModel;
-import de.unijena.cheminf.mortar.model.fragmentation.algorithm.ErtlFunctionalGroupsFinderFragmenter;
-import de.unijena.cheminf.mortar.model.fragmentation.algorithm.IMoleculeFragmenter;
-import de.unijena.cheminf.mortar.model.fragmentation.algorithm.ScaffoldGeneratorFragmenter;
-import de.unijena.cheminf.mortar.model.fragmentation.algorithm.SugarRemovalUtilityFragmenter;
+import de.unijena.cheminf.mortar.model.fragmentation.algorithm.*;
```

Comment:
Revert wildcard import as discussed (even though this is a very special case where this might almost make sense...)